packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -156,7 +266,7 @@ class _MyHomePageState extends State<MyHomePage>
                 // Usually it would be easier to use a builder from
                 // AdaptiveScaffold for these types of navigations but this
                 // navigation has custom staggered item animations.
-                child: NavigationRail(
+                child: AdaptiveScaffold.toNavigationRail(
```

Comment:
This does not need to be wrapped in a sizedBox anymore if you're using the builder since the builder already has the NavRail wrapped in a SizedBox right?